config-petro-tech.yaml

```diff
@@ -28,12 +28,12 @@ extractor:
 
 
 # subscriptions to stream
-subscriptions:
-  - external_id: gauge-ts
-    partitions:
-        - 0
-    lakehouse_abfss_path_dps: abfss://a0fd605a-3a69-401e-a586-bee3bf2c8127@onelake.dfs.fabric.microsoft.com/40b75c4d-5ddd-47ff-bd81-8faadec1804b/Tables/Timeseries
-    lakehouse_abfss_path_ts: abfss://a0fd605a-3a69-401e-a586-bee3bf2c8127@onelake.dfs.fabric.microsoft.com/40b75c4d-5ddd-47ff-bd81-8faadec1804b/Tables/TimeseriesMetadata
+subscription:
+  external_id: gauge-ts
+  name: Gauge Timeseries
+  num_partitions: 1
+  lakehouse_abfss_path_dps: abfss://a0fd605a-3a69-401e-a586-bee3bf2c8127@onelake.dfs.fabric.microsoft.com/40b75c4d-5ddd-47ff-bd81-8faadec1804b/Tables/Timeseries
```

Comment:
I know you did not introduce this file, but does it make sense to have a config with filled out values

Reply:
Updated to use env variables